Packs/Wiz/Integrations/Wiz/Wiz_test.py

```diff
@@ -506,6 +534,16 @@ def test_get_filtered_issues_bad_arguments(mocker, capfd):
                         'INFORMATIONAL in upper or lower case.'
 
 
+def test_get_issue_bad_arguments(mocker, capfd):
```

Comment:
What happens if the issue_id is in the right format but does not exists?

Reply:
The response will be empty as designed. The validations before the function only check the type and not invoke the api